### PR TITLE
WW-318: Add new icons

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Added new icons for
* user badges (used on discussions and Community Page)
* fandom logo (just fandom heart)
* help icon (used on Community Page)

@Wikia/west-wing @Wikia/x-wing

### CODE REVIEW

File: assets/icons/badge-vstf.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="18" height="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M0 9c0-1.15 1.247-2.04 1.66-3.04.43-1.035.196-2.544.976-3.324.78-.78 2.29-.546 3.324-.975C6.96 1.248 7.85 0 9 0c1.15 0 2.04 1.247 3.04 1.66 1.035.43 2.544.196 3.324.976.78.78.546 2.29.975 3.324C16.752 6.96 18 7.85 18 9c0 1.15-1.247 2.04-1.66 3.04-.43 1.035-.196 2.544-.976 3.324-.78.78-2.29.546-3.324.975-1 .413-1.89 1.66-3.04 1.66-1.15 0-2.04-1.247-3.04-1.66-1.035-.43-2.544-.196-3.324-.976-.78-.78-.546-2.29-.975-3.324C1.248 11.04 0 10.15 0 9" id="b"/><filter x="-50%" y="-50%" width="200%" height="200%" filterUnits="objectBoundingBox" id="a"><feOffset dy="1" in="SourceAlpha" result="shadowOffsetOuter1"/><feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.1 0" in="shadowOffsetOuter1"/></filter></defs><g fill-rule="evenodd"><g><use filter="url(#a)" xlink:href="#b"/><use xlink:href="#b"/></g><g fill="#FFF"><path d="M7.667 5h2.667v8H7.667z"/><path d="M13 7.667v2.667H5V7.667z"/></g></g></svg>
```
Comment (kvas-damian) [inline assets/icons/badge-vstf.svg]: Can we get rid of ` xmlns:xlink` ?

**Comment by rogatty** [comment]
Please update the reference page with new icons.

**Comment by hakubo** [comment]
We won't block you! :)
the discussion is about whether we should have the same icon in two different sizes
not if they all should have the same size of viewbox

File: assets/icons/badge-vstf.svg (hunk 1 above)
Comment (lukaszkonieczny) [inline assets/icons/badge-vstf.svg]: `xmlns:xlink` is used in these svg's

**Comment by cormorando** [comment]
LG

**Comment by sebastianmarzjan** [comment]
👍

**Pull request merged**